scripts/ci_steps/codegen_sdk.sh

```diff
@@ -12,7 +12,6 @@ rm -rf Tests/Services/*
 ./gradlew --stop
 
 # Regenerate the SDK Package.swift with all services
-unset AWS_SWIFT_SDK_USE_LOCAL_DEPS
```

Comment:
Un-setting this env var is no longer needed when building `AWSSDKSwiftCLI` since it does not depend on the SDK anymore.